### PR TITLE
Landing page

### LINKED ISSUE #31: Top section Image & “Build with Frequency”
- [ ] Reactive included (Ultrawide Desktop, Regular Desktop, IPad, Mobile, …)
- [ ] Image block (the large white section top left) use placeholder svg

Placeholder svg: Image

image

image

### PR DESCRIPTION
**Pull request by saraswatpuneet**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context.

- Closes #31 

## Screenshot(s)

Including screenshots helps reviewers know what to expect.

### Desktop

image

### Mobile
image

## Type of change

Please delete options that are not relevant.
- [x] New Feature

# How to Test?

Please describe how to test that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

1. git clone the repository
2. `cd frequency-xyz && npm i`
3. `npm run dev` then got to `http://localhost:3000/docs`

# Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code & PR, particularly in hard-to-understand areas
- [ ] I have checked at all the breakpoints to make sure it works on all screen sizes

### CODE REVIEW

**Comment by saraswatpuneet** [comment]
image

**Comment by saraswatpuneet** [comment]
image

File: src/components/Sections/SectionDefaultWave.svelte [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div
+  class={`lg:sm-[url(/src/lib/assets/placeholder-portal.svg)] absolute left-0 h-full bg-left-bottom bg-no-repeat sm:-bottom-16 sm:max-h-[100px] sm:w-[10%] sm:bg-[url(/src/lib/assets/section3mobile.svg)] md:max-h-full md:w-full lg:bottom-0 lg:max-w-full`}
+></div>
```
Comment (saraswatpuneet) [inline src/components/Sections/SectionDefaultWave.svelte:3]: what should i change here to fit the image in on the side?

image

**Comment by shannonwells** [comment]
Not sure if you meant for this to be reviewed yet, but it does have a lot of issues below 1300px and below 900px width

**Comment by saraswatpuneet** [comment]
Thought mobile version is still not correct with respect to alignment 
Screenshot_20241031-094818

**Comment by saraswatpuneet** [comment]
Fits on mobile screen now
image

File: src/routes/docs/components/DocsBody.svelte [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    </SlideIn>
+    <SlideIn>
+      <p class="title-50 paragraph-spacing-left text-orange sm:max-w-[600px] md:max-w-[700px] lg:max-w-[800px]">
+        Better with You
+      </p>
+    </SlideIn>
+    <p class="body paragraph-spacing-left sm:max-w-[600px] md:max-w-[700px] lg:max-w-[800px]">
+      Everything here is open source. Our development roadmap is focussed on our mission of reclaiming the social graph
+      for the common good and is guided by our principles. Infrastructure for the Next Generation Decentralized Social
+      Internet.
```
Comment (JoeCap08055) [inline src/routes/docs/components/DocsBody.svelte:23]:
```suggestion
      Everything here is open source. Our development roadmap is focused on our mission of reclaiming the social graph
      for the common good and is guided by our principles. Infrastructure for the Next Generation Decentralized Social
      Internet.
```

File: src/routes/docs/components/DocsBody.svelte [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    </p>
+    <p class="body paragraph-spacing-left sm:max-w-[600px] md:max-w-[700px] lg:max-w-[800px]">
+      If this is your first visit, the DSNP whitepape is a good place to start. Where you go after depends on your
+      goals: the nagivation panel gives some useful entry points.
```
Comment (JoeCap08055) [inline src/routes/docs/components/DocsBody.svelte:27]:
```suggestion
      If this is your first visit, the DSNP whitepaper is a good place to start. Where you go after depends on your
      goals: the navigation panel gives some useful entry points.
```

**Review by JoeCap08055 — CHANGES_REQUESTED** [review]
Couple of typos in the content, but otherwise looks good!

**Review by JoeCap08055 — APPROVED** [review]

File: src/routes/docs/+page.svelte [hunk 4]
```diff
@@ -1 +1,30 @@
...
+</script>
+
+<div class="min-h-[calc(100vh-100px)]">
+  <div class="relative lg:max-w-[100vw] xl:max-w-page">
+    <div class="absolute top-0 z-50 w-full"><Header {logoHex} {navColor} bind:section /></div>
+    <div class={`relative w-[100vw] min-w-full sm:pt-[80px] lg:pt-[140px] vertical-lg:w-[100%]`}>
+      <DocsBody />
+    </div>
+  </div>
+</div>
```
Comment (wilwade) [inline src/routes/docs/+page.svelte:30]: I think you can get away with leaving `src/routes/docs/+layout.svelte` alone and just doing this:

```suggestion
<div>
  <DocsBody />
</div>
```

Then you can also remove all the other section, color, and such.

**Review by wilwade — CHANGES_REQUESTED** [review]

File: src/routes/docs/+page.svelte (hunk 4 above)
Comment (saraswatpuneet) [reply src/routes/docs/+page.svelte]: I cannot get away with all, it then does not align with landing page, plus header is needed. I remove an outer class which had no effect

Comment (wilwade) [reply src/routes/docs/+page.svelte]: You can reset `src/routes/docs/+layout.svelte` and have the header in there right?

**Review by wilwade — APPROVED** [review]

**Pull request merged**